linode/manager
# feat: [M3-7374] - Add AGLB Configuration Create Flow

## Description 📝
- Allows users to create AGLB configurations 
- Allows users to remove routes from configurations rather than just outright deleting the route

## Preview 📷

![Screenshot 2023-11-03 at 1 07 01 PM](https://github.com/linode/manager/assets/115251059/aa15bc26-0d0d-4930-804c-9f4aa9bfb536)

## How to test 🧪

### Prerequisites
- Turn on the MSW
- Go to `http://localhost:3000/loadbalancers/0/configurations`

### Verification steps 

> [!important]
> Because we are testing with the MSW, some actions may have unexpected results

- Verify you can still **edit** a configuration
- Verify you can **create** a configuration

```
yarn cy:run -s "cypress/e2e/core/loadBalancers/load-balancer-configurations.spec.ts"
```

## As an Author I have considered 🤔
